src/SFA.DAS.EmployerIncentives.Domain/Interfaces/IIncentivePaymentProfilesService.cs

```diff
@@ -6,6 +6,6 @@ namespace SFA.DAS.EmployerIncentives.Domain.Interfaces
 {
     public interface IIncentivePaymentProfilesService
     {
-        Task<IEnumerable<IncentivePaymentProfile>> Get();
+        Task<IncentivesConfiguration> Get();
```

Comment:
Add a new Get overload  Task<IncentivePaymentProfile> Get(Phase phase) so the details can be loaded for the application/incentive if required.

Reply:
we did not intend to store Phase Id in this user story